new/LIT7101QeneAmh.xml

```diff
@@ -0,0 +1,100 @@
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <listWit>
+                    <witness corresp="UppQ1524o"/>
+                </listWit>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+                <abstract>
+                    <p>Amharic <foreign xml:lang="gez">qǝne</foreign> without type indication in the manuscript. It has eight lines, meaning
+                        that it can be classified as a <foreign xml:lang="gez">mawaddəs qəne</foreign> according to
+                        <bibl><ptr target="bm:Guidi1900Qene"/></bibl>.
+                        It is attributed to <persName ref="PRS14574GabraLeul" cert="high" corresp="SD">Gabra Ləʿul</persName> 
+                        in the manuscript.</p>
+                </abstract>
+            <textClass>
+                <keywords>
+                    <term key="AmharicLiterature"/>
+                    <term key="ChristianLiterature"/>
+                    <term key="Poetry"/>
+                    <term key="Qene"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="JK" when="2024-10-14">Created entity with the help of Hewan Semon Marye and Nafisa Valieva</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography">
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="LIT7101QeneAmh" passive="UppQ1524o"/>
+                    <relation name="saws:isAttributedToAuthor" active="LIT7101QeneAmh" passive="PRS14574GabraLeul"/>
+                </listRelation>
+            </div>
+            <div type="edition" xml:lang="am">
+                <note>This edition is based on <ref type="mss" corresp="UppQ1524o"/> on fols 45r, 46r.
+                    Division into verses is indicated with ፤ .
+                    The end of the poem is indicated with ። .</note>
+                <ab>
+                    <l n="1"><choice><sic>ኢ</sic><corr resp="HSM">እየ</corr></choice>ደ<choice><sic>ርስበ</sic><corr resp="HSM">ረሰብ</corr></choice>ኝ፡ 
```

Comment:
I propose to make one sic in one word even though, it is two unrelated phonetic questions.

Reply:
Is there a reason why that is better? (So that I remember next time)

Reply:
I hypothesise that this will look nicer on the website as well as in Oxygen, i.e. it will be easier to understand for anyone who reads it.